core/src/main/java/org/elasticsearch/search/aggregations/bucket/significant/SignificantTermsAggregatorFactory.java

```diff
@@ -293,7 +294,7 @@ public static ExecutionMode fromString(String value) {
                     return mode;
                 }
             }
-            throw new IllegalArgumentException("Unknown `execution_hint`: [" + value + "], expected any of " + values());
+            throw new IllegalArgumentException("Unknown `execution_hint`: [" + value + "], expected any of " + Arrays.toString(values()));
```

Comment:
These are unfortunate because these show up in client error responses. 😢

What's up with the backticks in the message though, this is not markdown?